ydb/docs/en/core/reference/ydb-cli/workload-transfer.md

```diff
@@ -112,6 +115,7 @@ Window  Write speed     Write time      Inflight        Lag             Lag time
 * `Lag` — the specified percentile of maximum number of messages waiting to be read in the statistics collection window. Messages for all batches are taken into account.
 * `Lag time` — the specified percentile of message delay time in ms.
 * `Read speed` — the speed of reading messages by consumers. In messages per second and in megabytes per second.
+* `Select time`, `Upsert time`, `Commit time` — the specified percentile of the execution time of Select, Insert, Commit operations in ms.
```

Comment:
We don't see this columns in the table above. It may be OK, just noticed.

Reply:
> We don't see this columns in the table above. It may be OK, just noticed.
✅